DeveloperDocumentation/Deployment/deployment-local.md

```diff
@@ -62,7 +62,24 @@ curl -H "Authorization: bearer <id token>" http://localhost:8091/whoami
 and some details about that user should be shown.
 
 ### Prepare authentication for fetching GitHub packages
-https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#authenticating-to-github-packages
+1. Navigate to the root directory of your machine, then do the following:
```

Comment:
The root directory could be misconstrued as `/`. Might be better to highlight that its the user's home directory.

Reply:
This is a good spot, I've changed the instruction in https://github.com/viabledata/ndtpia-integration-architecture-documentation/pull/1/commits/6b36573087c1e93d409bf4d9dbe5775fa5821e94